### DISCUSSION TITLE
Scribble Frontend Layout

### DISCUSSION THREAD

**Discussion by thelf** [Webmasters-Europe/codemeetup-forum#106] [General] [answered]
Analog zu den Backend Layouts werden auch Frontend Layouts benötigt.

- Ideen Sammlung der benötigten Elemente im Frontend
- Erstellung der Start- und Unterseiten als Scribble
- Umsetzung des Scribbles als klickbarer Prototyp

Das Frontend sollte ebenfalls responsive sein.

Vorschlag für Open Source Software für die Erstellung von Scribbles (Windows, Mac und Ubuntu wird unterstützt) https://pencil.evolus.vn/

**Comment by alenavershinina** [comment, marked as answer]
Ich habe ein paar Scribbles fuer Large Screen gemacht.
Ich bin nicht sicher, ob wir Navigation Bar brauchen. Ich schlage vor, alles in Side Bar zu stellen.
![forum](https://user-images.githubusercontent.com/74716234/107061643-b7d94b00-67d8-11eb-9cf2-dd8c9f73201f.png)
![logged_in](https://user-images.githubusercontent.com/74716234/107061645-b871e180-67d8-11eb-8064-ca919feb1b67.png)
